### DISCUSSION TITLE
Can we have just the main progress bar?

### DISCUSSION THREAD

**Discussion by joaociocca** [epi052/feroxbuster#701] [Ideas]
I think I mentioned this before on twitter, and you were awesome talking to me about it over there about it... but srsly, would be really nice to have just the main progress bar. Or be able to set a limit on progress bars being shown. Or hide away completed progress bars.

![image](https://user-images.githubusercontent.com/6473725/201539900-8a487815-1b2e-45a1-b344-954a5bfd4ffa.png)

Specially because ferox and tmux  have some kinda beef when the progress bars go beyond the pane size, it just goes crazy repeating the same bars =p

![image](https://user-images.githubusercontent.com/6473725/201539978-5d1b56b1-8acc-4f49-9e9a-28bd4ec91864.png)

![image](https://user-images.githubusercontent.com/6473725/201540003-6ff2e7d7-b2e1-424a-b774-ebd40135f73b.png)

**Comment by joaociocca** [comment]
hah, I just seen this suggestion! 

https://github.com/epi052/feroxbuster/issues/663

Seems to kinda align with my case =p

**Comment by joaociocca** [comment]
Even `-q` does this...

![image](https://user-images.githubusercontent.com/6473725/201540234-da09a7d6-7514-4297-84c7-b648fbdda021.png)

It starts kinda nicely... but then... when it hits the top of the pane...

![image](https://user-images.githubusercontent.com/6473725/201540267-dbe77e05-1fda-4e13-a918-458b7e3ea83f.png)

**Comment by epi052** [comment]
hey there! I briefly looked at what it would take to add another bar configuration. I'm not planning on adding a whole new option to ferox rn, but if it's simple, I may expose an environment variable or something similar to limit output to just the overall scan bar. 

Not sure just yet, but wanted to let you know. And yea, the linked issue seems to want roughly the same thing :grin:
